src/panels/config/zone/dialog-core-zone-detail.ts

```diff
@@ -0,0 +1,278 @@
+          name="name"
+          .label=${this.hass.localize(
+            "ui.panel.config.core.section.core.core_config.location_name"
+          )}
+          .disabled=${disabled}
+          .value=${this._name}
+          @change=${this._handleChange}
+        ></ha-textfield>
+        <ha-select
+          .label=${this.hass.localize(
+            "ui.panel.config.core.section.core.core_config.time_zone"
+          )}
+          name="timeZone"
+          fixedMenuPosition
+          naturalMenuWidth
+          .disabled=${disabled}
+          .value=${this._timeZone}
+          @closed=${stopPropagation}
+          @change=${this._handleChange}
+        >
+          ${Object.keys(timezones).map(
+            (tz) =>
+              html`<mwc-list-item value=${tz}>${timezones[tz]}</mwc-list-item>`
+          )}
+        </ha-select>
+        <ha-textfield
+          .label=${this.hass.localize(
+            "ui.panel.config.core.section.core.core_config.elevation"
+          )}
+          name="elevation"
+          type="number"
+          .disabled=${disabled}
+          .value=${this._elevation}
+          @change=${this._handleChange}
+        >
+          <span slot="suffix">
+            ${this.hass.localize(
+              "ui.panel.config.core.section.core.core_config.elevation_meters"
+            )}
+          </span>
+        </ha-textfield>
+        <div>
+          <div>
+            ${this.hass.localize(
+              "ui.panel.config.core.section.core.core_config.unit_system"
+            )}
+          </div>
+          <ha-formfield
+            .label=${html`${this.hass.localize(
+                "ui.panel.config.core.section.core.core_config.unit_system_metric"
+              )}
+              <div class="secondary">
+                ${this.hass.localize(
+                  "ui.panel.config.core.section.core.core_config.metric_example"
+                )}
+              </div>`}
+          >
+            <ha-radio
+              name="unit_system"
+              value="metric"
+              .checked=${this._unitSystem === "metric"}
+              @change=${this._unitSystemChanged}
+              .disabled=${this._submitting}
+            ></ha-radio>
+          </ha-formfield>
+          <ha-formfield
+            .label=${html`${this.hass.localize(
+                "ui.panel.config.core.section.core.core_config.unit_system_imperial"
+              )}
+              <div class="secondary">
+                ${this.hass.localize(
+                  "ui.panel.config.core.section.core.core_config.imperial_example"
+                )}
+              </div>`}
+          >
+            <ha-radio
+              name="unit_system"
+              value="imperial"
+              .checked=${this._unitSystem === "imperial"}
+              @change=${this._unitSystemChanged}
+              .disabled=${this._submitting}
+            ></ha-radio>
+          </ha-formfield>
+        </div>
+        <div>
+          <ha-select
```

Comment:
We previously had this with a data element such that it would autocomplete in the browsers.

Reply:
Auto Complete Currency? hmm It was using the old paper elements. Ill see if there is an autocomplete for mwc?

Reply:
it was using a [`<datalist>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) which we attached to the `<input>` inside the paper element.

Reply:
Same in timezones btw.

Having a combobox is probably better, not sure why we didn't use that.